src/styles/index.scss

```diff
@@ -0,0 +1,6 @@
+/**
+ * Please use rem for styling (Desktop size: 1rem = 10px)
+ */
+
+@import './base/base';
+@import './base/typography';
```

Comment:
If this is the root SCSS file then should we import all modules such as `vendor`, `abstracts`...

Reply:
No need to import abstract to root

Reply:
I'll leave one comment to clarify we need to import vendor here

Reply:
Why shouldn't we import abstract to the root then we need to import it in stylesheet that we want to use?